tests/Unit.Tests/Controllers/CourseController.Tests.cs

```diff
@@ -0,0 +1,81 @@
+
+
+using System;
+using System.Collections.Generic;
+using GovUk.Education.SearchAndCompare.Domain.Client;
+using GovUk.Education.SearchAndCompare.Domain.Models;
+using GovUk.Education.SearchAndCompare.UI.Controllers;
+using GovUk.Education.SearchAndCompare.UI.ViewModels;
+using Microsoft.AspNetCore.Mvc;
+using Moq;
+using NUnit.Framework;
+
+namespace GovUk.Education.SearchAndCompare.UI.Unit.Tests.Controllers
+{
+    [TestFixture]
+    public class CourseControllerTests
+    {
+        private Mock<ISearchAndCompareApi> mockApi;
+        private CourseController sut;
+
+        [SetUp]
+        public void SetUp()
+        {
+            mockApi = new Mock<ISearchAndCompareApi>();
+            sut = new CourseController(mockApi.Object);
+        }
+
+        [Test]
+        public void Index_CallsApiCorrectly()
+        {
+            mockApi.Setup(x => x.GetCourse("XYZ", "1AB")).Returns(GetSimpleCourse()).Verifiable();
+            mockApi.Setup(x => x.GetFeeCaps()).Returns(GetFeeCaps()).Verifiable();
+
+            var actionResult = sut.Index("XYZ", "1AB", new Filters.ResultsFilter());
+
+            Assert.That(actionResult is ViewResult);
+            Assert.That((actionResult as ViewResult).Model is CourseViewModel);
+
+            var courseViewModel = (actionResult as ViewResult).Model as CourseViewModel;
+
+            Assert.AreEqual("My course", courseViewModel.Course.Name);
+            Assert.AreEqual(200, courseViewModel.Finance.FeeCaps.InternationalFees);
```

Comment:
Use a field const to make it clearer that it matters that the two 200 numbers across the test class are related.

Minor